{{cookiecutter.github_repository}}/{{cookiecutter.main_module}}/urls.py

```diff
@@ -44,6 +48,9 @@
     ),
     # Rest API
     path("api/", include(api_urls)),
+{%- if cookiecutter.add_graphql == "y" %}
+    path("graphql/", GraphQLView.as_view(graphiql=True)),
```

Comment:
Let's pass `GRAPHQL_DEBUG` value in `graphiql=` here, so that the graphiql interface is only visible on dev environment.